tests/events/test_presence_router.py

```diff
@@ -162,7 +162,10 @@ def prepare(self, reactor, clock, homeserver):
                     },
                 }
             },
-            "send_federation": True,
+            # Default test case disables federation sending. Setting
+            # 'federation_sender_instances' to None turns it back on for the main
+            # process
+            "federation_sender_instances": None,
```

Comment:
Is this enabled for every test in this class? Can we put this in `default_config`, if so?

Reply:
I'll check that it is for every test, but I believe so. Frankly, I didn't know enough about how `default_config` was processed to mess with that. So I just fixed it in place where it was before. Consolidation though, I'm on board with that.